src/schematools/types.py

```diff
@@ -110,9 +110,9 @@ def license(self) -> Optional[str]:
         return self.get("license")
 
     @property
-    def url_prefix(self) -> str:
-        """Dataset URL prefix"""
-        return self.get("url_prefix", "")
+    def path(self) -> str:
+        """Dataset URL path"""
+        return self.get("path", "")
```

Comment:
Would not it be better to set path default to `id`? Then we won't need extra mandatory parameter in schemas, that in 99% of cases is equal to ID.

Reply:
Wouldnt that be tricky with maintaining uniqueness? If some dataset_a.path == dataset_b.id

Reply:
I thought about parsing path from id. So id would be something like "haalcentraal__brk" and we could just convert that to "haalcentraal/brk" But then you might run into the 63 length limit for tables.

Reply:
But that's same story as we have now with `path` specified on all schemas, isn't it? 

Migration in this PR uses `id` as `path` and in cases where Dataset needs to "cover" another id with url - developers can set `path` on one or both datasets in order to achieve desired result.

Reply:
>I thought about parsing path from id. So id would be something like "haalcentraal__brk" and we could just convert that to "haalcentraal/brk" But then you might run into the 63 length limit for tables

In addition to 63 chars limits it will produce whole set of annoying snake_case to camelCase issues. :)

What I meant by path - is to make it default to id, but allow schemas to define path if needed. Hope this makes my comment a little bit more clear.